### DISCUSSION TITLE
Exclude a Species from Daily Chart?

### DISCUSSION THREAD

**Discussion by enthusiastpodcast** [mcguirepr89/BirdNET-Pi#1197] [Q&A]
Something I've learned quickly from having my BirdNET Pi running for a few weeks is that a number of Wood Thrushes live in my backyard and sing all day. I've always noticed them before, but I didn't realize just how disproportionate their song was to other species: 
![Combo-2024-05-05](https://github.com/mcguirepr89/BirdNET-Pi/assets/64302004/24faed8c-6499-40bc-b42c-982461800ba4)

3000+ Wood Thrush detections is awesome, but makes the rest of the chart impossible to read lol. Is there a way to keep Wood Thrush from clogging this chart, but still detect the birds? Or maybe to set the chart to use a log scale or something for frequency?

Thanks,
Mark

**Comment by thestig4242** [comment]
I just started a couple days ago, and I love the project! I'm having the same issue but with flycatchers. They chirp at least every 3 seconds in the morning so they almost always get identified instead of any other bird. If I dive into the model and figure out a way to put a cool down time on certain species I'll report back.

**Reply by thestig4242** [reply]
I did a quick hack to get the top ten to plot frequency in logarithmic scale. 
In the folder scripts, edit the file: daily_plot.py 

On line 83, or on a blank line right below: plot.set(xlabel="detections)

Add: 
plot.set(xscale = 'log')
